.tekton/cli-main-ci-pull-request.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
       == "main"
-  creationTimestamp: null
```

Comment:
Are we confident that removing `creationTimestamp: null` is fine?

Reply:
I've done it in other places without any issues.